_content/articles/dharma-songs-in-contemporary-cambodia_walker-trent.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Saṃvega and Pasāda: Dharma Songs in Contemporary Cambodia"
+authors:
+  - "Trent Walker"
+external_url: "https://poj.peeters-leuven.be/content.php?url=article.php&id=3285745&journal_code=JIABS"
+drive_links:
+  - "https://drive.google.com/file/d/1OjKQZ3i9r38MdEQM-cttpFf2CADnzf3Z/view?usp=sharing"
+course: cambodia
+tags:
+  - music
+  - bart
```

Comment:
Yeah, bart is good. Especially as it contains musings on Buddhist aesthetics as a whole